install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,206 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
+sample use-case though this method can be adapted to any
+link:../install_config/persistent_storage/index.html[supported storage plugin].
+
+=== Prerequisites
+* A gluster volume exists and *glusterfs-fuse* is installed on all hosts
```

Comment:
(Comparing with style in https://docs.openshift.com/enterprise/latest/install_config/persistent_storage/persistent_storage_glusterfs.html) Init cap "Gluster" throughout (or "GlusterFS" as appropriate).